tests/WebAssertTest.php

```diff
@@ -982,6 +982,81 @@ public function testElementAttributeExists()
         );
     }
 
+    public function testElementAttributeNotExistsValid()
+    {
+        $page = $this->getMockBuilder('Behat\\Mink\\Element\\DocumentElement')
+            ->disableOriginalConstructor()
+            ->getMock()
+        ;
+
+        $element = $this->getMockBuilder('Behat\\Mink\\Element\\NodeElement')
+            ->disableOriginalConstructor()
+            ->getMock()
+        ;
+
+        $this->session
+            ->expects($this->exactly(1))
+            ->method('getPage')
+            ->will($this->returnValue($page))
+        ;
+
+        $page
+            ->expects($this->exactly(1))
+            ->method('find')
+            ->with('css', 'h2 > span')
+            ->will($this->returnValue($element))
+        ;
+
+        $element
+            ->expects($this->exactly(1))
+            ->method('hasAttribute')
+            ->with('blah')
+            ->will($this->returnValue(false))
+        ;
+
+        call_user_func_array(array($this->assert, 'elementAttributeNotExists'), array('css', 'h2 > span', 'blah'));
+    }
+
+    public function testElementAttributeNotExistsThrowsException()
```

Comment:
Not done. There are no such `testElementAttributeExistsThrowsException` test.

Since tested method is a copy paste from `elementAttributeExists` method I expect same throws checking test will exist for it already and you'll just copy paste yours from it.